things/calculator/http/express/main.js

```diff
@@ -91,17 +91,25 @@ app.post(`/${thingName}/${ACTIONS}/subtract`, reqParser, (req, res) => {
   }
 })
 
-app.get(`/${thingName}/${EVENTS}/change`, (req, res) => {
+//*Changed the endpoint to "/update" instead of "/change" as thats how it is specified in the TD
```

Comment:
I don't think this comment is useful since it was a "bug" before

Reply:
Sorry about that I meant to remove the comment before the PR.
But just to be sure you mean just the comment right?, Or are you saying that the endpoint name should not be changed to "update"?

Reply:
> But just to be sure you mean just the comment right?

Yes, the comment should not be there,

`/update`  is fine
https://github.com/eclipse-thingweb/test-things/blob/ca67624c1ffac7324606a1aaf9556d291b6c4e38/things/calculator/calculator.tm.json#L101